tests-functional/pages/all_aboard.py

```diff
@@ -3,13 +3,44 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from pypom import Page
+
 from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
 
 
 class AllAboard(Page):
 
-    _sidebar_locator = (By.CSS_SELECTOR, '#content-deck #sidebar-box')
+    _action_button_locator = (
+        By.ID,
+        'action-button--all-aboard-v1-all-aboard'
+    )
+    _active_notification_locator = (
+        By.CSS_SELECTOR,
+        '#action-button--all-aboard-v1-all-aboard[badge="1"]'
+    )
+    _sidebar_locator = (
+        By.CSS_SELECTOR,
+        '#content-deck #sidebar-box'
```

Comment:
Is this an overly specific selector as ID's should be unique? Could you just use `_sidebar_locator = (By.ID, 'sidebar-box')`?